homeassistant/components/zwave_js/lock.py

```diff
@@ -55,6 +63,26 @@ def async_add_lock(info: ZwaveDiscoveryInfo) -> None:
         )
     )
 
+    platform = entity_platform.current_platform.get()
+    assert platform
+
+    platform.async_register_entity_service(  # type: ignore
```

Comment:
What's the type error we ignore here?

Reply:
`homeassistant/components/zwave_js/lock.py:78: error: Call to untyped function "async_register_entity_service" of "EntityPlatform" in typed context  [no-untyped-call]
Found 1 error in 1 file (checked 1 source file)`